.fixtures.yml

```diff
@@ -3,9 +3,13 @@ fixtures:
     apache: 'https://github.com/puppetlabs/puppetlabs-apache'
     apt: 'https://github.com/puppetlabs/puppetlabs-apt'
     augeas_core: 'https://github.com/puppetlabs/puppetlabs-augeas_core'
+    certs: 'https://github.com/theforeman/puppet-certs'
     concat: 'https://github.com/puppetlabs/puppetlabs-concat'
     cron_core: 'https://github.com/puppetlabs/puppetlabs-cron_core'
     extlib: 'https://github.com/voxpupuli/puppet-extlib'
+    hashfile: 'https://github.com/southalc/hashfile'
+    iop_advisor_engine: 'https://github.com/theforeman/puppet-iop_advisor_engine'
```

Comment:
Should `iop_advisor_engine` become a real dependency of the foreman module, or "soft", given we only need it in the rh cloud case?

Reply:
https://github.com/theforeman/foreman-installer/pull/1005